### ISSUE TITLE
Error Compiling

### ISSUE THREAD

**Issue by Briano99** [sabinbajracharya/fluttery-todo#1]
Resolving dependencies...
* Error running Gradle:
ProcessException: Process "C:\Users\boh5\flutter-dev\fluttery-todo\android\gradlew.bat" exited abnormally:

FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Users\boh5\flutter-dev\fluttery-todo\android\app\build.gradle' line: 54

* What went wrong:
A problem occurred evaluating project ':app'.
> path may not be null or empty string. path='null'

**Comment by ifdotpy** [comment]
@Briano99 replace file `android/app/build.gradle` with https://gist.github.com/IFLead/5e4700111d1ae3bb99c3e3ac360c712c

**Comment by sabinbajracharya** [comment]
You can either create **key.properties** file inside android directory with the following content:
```
storePassword=yourPassword
keyPassword=yourPassword
keyAlias=key
storeFile=C:/todokey.jks
```

Replace the values with yours (Note: You need to create a signing key).

Or as @IFLead  mentioned you could  open **build.gradle** inside app folder and replace 

 ```
signingConfigs {
        release {
            keyAlias keystoreProperties['keyAlias']
            keyPassword keystoreProperties['keyPassword']
            storeFile file(keystoreProperties['storeFile'])
            storePassword keystoreProperties['storePassword']
        }
    }
    buildTypes {
        release {
            signingConfig signingConfigs.release
            minifyEnabled true
            useProguard true
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
}
```

with:

```
buildTypes {
        release {
            // TODO: Add your own signing config for the release build.
            // Signing with the debug keys for now, so `flutter run --release` works.
            signingConfig signingConfigs.debug
        }
}
```

**Comment by sabinbajracharya** [comment]
fixed #3
